state_machine/applications/flight/Tasks/log.py

```diff
@@ -0,0 +1,61 @@
+from lib.template_task import Task
+from pycubed import cubesat
+import logs
+import traceback
+
+log_fd = None
+log_fd_str = None
+
+class Task(Task):
```

Comment:
Renaming `Task` seems sketchy to me. I suppose it works, but naming this `LogTask` and then when it's imported doing `from Tasks.log_task import LogTask as Task` is much clearer